src/olympia/addons/tests/test_admin.py

```diff
@@ -497,20 +497,24 @@ def test_query_count(self):
         self.grant_permission(user, 'Addons:Edit')
         self.grant_permission(user, 'Admin:Advanced')
         self.client.login(email=user.email)
-        with self.assertNumQueries(26):
+        with self.assertNumQueries(20):
```

Comment:
Can we have a test that confirms `assertNumQueries` for the list view?

Reply:
That's my FIXME below - it will still be terrible while https://github.com/mozilla/addons-server/issues/16132 isn't fixed (next on my todo list) but I'll add that yeah.

Reply:
e99376e1578c0b9a1f2eb6f8aaf1d059aa5ca05b